shubhali/HTML_File/HTML_Que&Ans

```diff
@@ -0,0 +1,61 @@
+ACTIVITY1:
+Q1.What did you find most intresting about the history of HTML?
+->Rather than inventing new standard ,Sir Tim used alredy existed tag in SGML.
+
+
+ACTIVITY2:
+Q1.Which is tag and which is attribute in the following line?
+   <div id="footer">
+->tag-div, attribute-id.
+
+Q2.Why is HTML called a mark-up language?
+->Hypertext means machine readable text and Markup means to structure it in a specific format.So,HTML is called hypertext markup language because it is a language that allows users to organize, improve the appearance of, and link text with data on the internet.
+
+Q3.Describe one problem you wouuld likely face if you included presentation information with HTML instead of separationg it in css?
+->we have to add attribute for each tag.time consuming process.
```

Comment:
need to describe this ans

Reply:
if you write separate css file then also you have to add attribute you have to write ans for if you are not writing separate css file then what kind of problem you might face?